src/wp-includes/block-template.php

```diff
@@ -17,6 +17,41 @@ function _add_template_loader_filters() {
 	}
 }
 
+/**
+ * Renders a warning screen for empty block templates.
+ *
+ * @since x.x
+ *
+ * @param WP_Block_Template $block_template The block template object.
+ * @return string The warning screen HTML.
+ */
+function wp_render_empty_block_template_warning( $block_template ) {
+	wp_enqueue_style( 'wp-empty-template-alert' );
+	return sprintf(
+		/* translators: %1$s: Block template title. %2$s: Empty template warning message. %3$s: Edit post link. %4$s: Edit button label. */
+		'<div id="wp-empty-template-alert">
+			<div class="icon">
```

Comment:
this icon does not add value; let's just remove it.

Reply:
Done in https://github.com/WordPress/wordpress-develop/pull/7354/commits/1792a47b395e70c5a17654a1be56306e11e1eb02.